README.md

```diff
@@ -1 +1,28 @@
 # ToDoAppFastApi
+This app contains a demo To-Do app using the following:
+1) Fast API
+2) Postgresql
+3) RabbitMQ
+4) Flower
+5) PgAdmin
+6) Docker
+7) Docker Compose
+8) Celery
+9) Celery Beat
+
+-----------------------------------------------------------
+# Run the app
+To run the app follow below given steps:
+- Go to terminal
+- Add a new network called "frcp"
+  - docker network create frcp
```

Comment:
these steps should not be needed.. can you remove networks from docker compose and try again.. if it works without netwoks, update steps here then